src/components/IPScan/Summary/index.js

```diff
@@ -342,20 +342,6 @@ const SummaryIPScanJob = ({
               <tr>
                 <td>Title</td>
                 <td style={{ display: 'flex' }}>
-                  {title}{' '}
-                  <Tooltip title={'Edit Title'}>
-                    <button
-                      className={f(
-                        'icon',
-                        'icon-common',
-                        'small',
-                        'button-space',
-                      )}
-                      data-icon="&#xf303;"
-                      aria-label={'Edit Title'}
-                      onClick={() => setShowTitleField(!showTitleField)}
-                    />
-                  </Tooltip>
                   {showTitleField ? (
                     <div className={f('title-form')}>
                       <input
```

Comment:
Can you prepopulate this field with the current name. it might be the case the user just want edit a current name

Reply:
yeah, Sure. I will make the change